doc/uftrace-record.md

```diff
@@ -375,16 +376,11 @@ The `traceon` and `traceoff` actions (the `_` can be omitted from `trace_on`
 and `trace_off`) control whether uftrace records the specified functions or not.
 
 The 'recover' trigger is for some corner cases in which the process accesses the
-callstack directly.  During tracing of the v8 javascript engine, for example, it
-kept getting segfaults in the garbage collection stage.  It was because v8
-incorporates the return address into compiled code objects(?).  The `recover`
-trigger restores the original return address at the function entry point and
-resets to the uftrace return hook address again at function exit.  I was managed
-to work around the segfault by setting the `recover` trigger on the related
-function (specifically `ExitFrame::Iterate`).
+callstack directly.  For now it's not necessary to call it as uftrace does the
+job automatically.
```

Comment:
This applies to x86 only.  It's hard to do the same on ARM/AArch64 at least currently..